sqltocsv.go

```diff
@@ -51,7 +51,10 @@ func main() {
 			if err != nil {
 				log.Fatal(err)
 			}
-			csvWriter.Write(cols)
+			write, err := csvWriter.Write(cols)
```

Comment:
When I `make build` locally I get the following error:
```
./sqltocsv.go:54:15: assignment mismatch: 2 variables but 1 values
```
My go knowledge is very limited but I suspect you need to call `write` on L58?

Reply:
I was under the impression the write class had a built in error return, but I may have misread the documentation. Will debug further.